src/setup-beam.js

```diff
@@ -184,13 +185,16 @@ async function getElixirVersion(exSpec0, otpVersion0, hexMirrors) {
   const otpVersion = otpVersion0.match(/^([^-]+-)?(.+)$/)[2]
   const otpVersionMajor = otpVersion.match(/^([^.]+).*$/)[1]
   const elixirVersions = await getElixirVersions(hexMirrors)
-  const semverVersions = Array.from(elixirVersions.keys()).sort()
```

Comment:
This is now done inside `getVersionFromSpec/2`, for all consumer functions, as to centralised behaviour and make it more consistent.